src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -195,7 +195,7 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getJavaToolsJar() {
+  public Path getJavaToolsJar() {
```

Comment:
@loosebazooka it's being used for a few things in the Eclipse plugin.
Find shared/servlet-api.jar and shared/jsp-api.jar, display the current location on the text box, set it for internal use, check that the path exists (though this should really be being done at the library), etc.